tests/support/Traits/TableTestStateMockTrait.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace RonasIT\Support\Tests\Support\Traits;
+
+use Illuminate\Database\Query\Builder;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\DB;
+
+trait TableTestStateMockTrait
```

Comment:
`TableTestStateMockTrait` and `ModelTestStateMockTrait` are very similar. Can we combine them?